components/Modals/SignUp.tsx

```diff
@@ -12,6 +12,9 @@ import { generateKeypair } from 'lib/crypto'
 import { register } from 'api/auth'
 import { Input } from 'shared'
 import Modal from './Modal'
+{
```

Comment:
@sabakhilji can you uncomment the code in SignUp.tsx so that the component is visible for testing in deployment link